README.md

```diff
@@ -226,8 +226,10 @@ Options, and their defaults
   // the port the app will start on
   port: 8080,
   // whether or not to run in parallel using all available cpus
-  endpoint: '/batch'
+  endpoint: '/batch',
   // default endpoint path
+  // return the number of cpus hypernova will use
+  getCPUs: undefined
```

Comment:
Let me know if the new text looks good (unsure exactly how much detail you're looking for here since I documented it further down in the readme. maybe we can kill that though if it's fully documented here instead?)